research-hub-web/src/styles.scss

```
@@ -216,6 +216,12 @@ app-search-filters {
   }
 }
 
+mat-option .mat-option-text {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
```

Given this is in a global style it would be good to scope it to the component that it appears in. Otherwise this could apply to any mat-option in the app

```suggestion
app-search-bar {
  mat-option .mat-option-text {
    overflow: hidden;
    white-space: nowrap;
    text-overflow: ellipsis;
  }
}
```

---

Fixed by adding a class to the mat-options. Thanks Lukas!